tcms/static/js/testrun_actions.js

```diff
@@ -683,6 +655,62 @@ jQ(document).ready(function(){
   });
 });
 
+function drawPercentBar() {
+
+  const testRunId = $('input[name="run_id"]').val()
+
+  jsonRPC('TestExecution.filter', {'run_id': testRunId}, testExecutions => {
+    
+    let positiveCount = 0;
+    let negativeCount = 0;
+    let allCount = testExecutions.length;
+    let statusCount = {}
+    executionStatuses.forEach(s => statusCount[s.name] = { count: 0, id: s.id })
+
+    testExecutions.forEach(testExecution => {
+      const executionStatus = executionStatuses.find(s => s.id === testExecution.status_id)
+
+      if (executionStatus.weight > 0) {
+        positiveCount++
+      } else if (executionStatus.weight < 0) {
+        negativeCount++
+      }
+
+      statusCount[executionStatus.name].count++
+    })
+  
+    const completeCount = positiveCount + negativeCount;
+    const completePercent = completeCount / allCount * 100;
+    const failurePercent = negativeCount / completeCount * 100;
```

Comment:
Causes a regression wrt to how many decimal points are shown. Limit to only 2 decimal points at most.